src/pages/api/user/login.ts

```diff
@@ -1,19 +1,54 @@
-// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
-import type { NextApiRequest, NextApiResponse } from 'next'
-
-export interface LoginRequestType {
-  name: string,
-}
-
-export interface LoginResponseType {
-  error?: string,
-}
-
-export default function handler(
-  req: NextApiRequest,
-  res: NextApiResponse<LoginResponseType>
-) {
-  
-
-  res.status(200).json({ error: 'John Doe' })
+import type { NextApiRequest, NextApiResponse } from "next";
+import { prisma } from '../../../lib/prisma'
+
+// API Inputs.
+export interface LoginRequestType {
+    firstName: string,
+    lastName: string,
+    email: string,
+    password: string,
+}
+
+// API Outputs.
+export interface LoginResponseType {
+    user?: Object,

```

Comment:
You can import the User model using `import { User } from '@prisma/client'`. Then type the user like `user?: User`